opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -125,7 +141,33 @@
         /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
         SpanBuilder withStartTimestamp(long microseconds);
 
-        /** Returns the started Span. */
+        /**
+         * Returns a newly started and {@linkshort Scheduler.Continuation#activate(boolean) activated}
```

Comment:
There is not `linkshort` Javadoc tag. Either use `link` or `linkplain`, if 'activated' should be displayed in plain text rather than in code font.